benchmarks/ekomark/benchmark/external/read_lha_pdf.py

```diff
@@ -5,12 +5,775 @@
+                1.3200e-4,
+                6.9268e-6,
+                1.3739e2,
+                6.7627e1,
+                6.5548e1,
+                5.5295e1,
+                9.4403e2,
+            ],
+            [
+                10e-6,
+                8.1001e-4,
+                5.3574e-4,
+                3.0345e-5,
+                7.2374e1,
+                3.5337e1,
+                3.3846e1,
+                2.7870e1,
+                4.7444e2,
+            ],
+            [
+                10e-5,
+                3.4428e-3,
+                2.1524e-3,
+                1.2531e-4,
+                3.5529e1,
+                1.7091e1,
+                1.6065e1,
+                1.2883e1,
+                2.1802e2,
+            ],
+            [
+                10e-4,
+                1.4580e-2,
+                8.6744e-3,
+                4.8276e-4,
+                1.6042e1,
+                7.4886e0,
+                6.8276e0,
+                5.3044e0,
+                8.9013e1,
+            ],
+            [
+                10e-3,
+                6.0912e-2,
+                3.5030e-2,
+                1.7393e-3,
+                6.5544e0,
+                2.8656e0,
+                2.4802e0,
+                1.8362e0,
+                3.0617e1,
+            ],
+            [
+                10e-2,
+                2.3327e-1,
+                1.3022e-1,
+                5.7588e-3,
+                2.2949e0,
+                8.6723e-1,
+                6.7688e-1,
+                4.5597e-1,
+                7.8243e0,
+            ],
+            [
+                0.1,
+                5.4798e-1,
+                2.6905e-1,
+                9.9470e-3,
+                3.8192e-1,
+                1.1124e-1,
+                6.7091e-2,
+                3.7698e-2,
+                8.4908e-1,
+            ],
+            [
+                0.3,
+                3.4291e-1,
+                1.2693e-1,
+                2.9239e-3,
+                3.4069e-2,
+                8.6867e-3,
+                4.3924e-3,
+                2.1435e-3,
+                7.8109e-2,
+            ],
+            [
+                0.5,
+                1.1694e-1,
+                3.0310e-2,
+                3.6112e-4,
+                2.2828e-3,
+                5.5537e-4,
+                2.7744e-4,
+                1.2416e-4,
+                7.5371e-3,
+            ],
+            [
+                0.7,
+                1.9076e-2,
+                2.9217e-3,
+                1.2635e-5,
+                5.2061e-5,
+                1.2677e-5,
+                7.2083e-6,
+                3.0908e-6,
+                3.6441e-4,
+            ],
+            [
+                0.9,
+                3.2404e-4,
+                1.6333e-5,
+                7.900e-9,
+                2.850e-8,
+                8.407e-9,
+                6.795e-9,
+                3.205e-9,
+                1.1411e-6,
+            ],
+        ],
+    ]
```

Comment:
Maybe this is not the most clever idea ever. If you have some data put them in a data file, being it `.yaml` or `.json` or whatever, and load it in the code, instead of embedding data in your source.

Reply:
From my side this was always intended as quick and dirty wrapper to cast the strings (from paper PDF) to python, I don't know if @giacomomagni used it the same way - if so, and the data is already in the `.yaml` simply remove the stuff from here again (or maybe delete the whole file, since there are no more paper data after NNLO)

Reply:
Hi, yes I agree with @felixhekhorn this script was just quick and dirty, it can be deleted.